.vscode/schema/manifest-shim.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
```

Comment:
Now we have an ad-hoc `iOS-injected-scripts` in our iOS build. We may add it here as:

```typescript
"userscripts": string[]
```